CampaignConfig/fclless/run_Primary.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
This looks like the script running on the worker node, could you add a comment explaining that?

Reply:
yes, this is ran on the grid. It should be moved to Scripts eventually